package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@enact/storybook-utils",
-  "version": "4.0.2",
+  "version": "4.0.3",
```

Comment:
Please leave this as-is. It's changed during releasing process.

Reply:
OK